benchmarks/mrbbagging/data/cleveland-sm.arff

```diff
@@ -0,0 +1,300 @@
+@relation cleveland-wothu4
```

Comment:
`benchmarks/` is not the best directory for storing data ;)

Reply:
Followed the convention from SPIDER, can compress it if necessary.